src/test/java/org/testeditor/fixture/host/s3279/StatusTest.java

```diff
@@ -0,0 +1,190 @@
+package org.testeditor.fixture.host.s3279;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.testeditor.fixture.host.exceptions.StatusNotFoundException;
+import org.testeditor.fixture.host.s3270.Status;
+import org.testeditor.fixture.host.s3270.options.TerminalMode;
+import org.testeditor.fixture.host.s3270.statusformat.ConnectionState;
+import org.testeditor.fixture.host.s3270.statusformat.EmulatorMode;
+import org.testeditor.fixture.host.s3270.statusformat.FieldProtection;
+import org.testeditor.fixture.host.s3270.statusformat.KeyboardState;
+import org.testeditor.fixture.host.s3270.statusformat.ScreenFormatting;
+
+public class StatusTest {
+
+  String defaultResult = "U F U C(abcdefg.hi.google-mainframe.com) I 2 24 80 6 44 0x0 -";
+
+  @Test
+  public void keyboardStateSuccessfullTest() {
+    String result = defaultResult;
+    Status status = new Status(result);
+    Assert.assertTrue(status.getKeyboardState().name().equals(KeyboardState.UNLOCKED.name()));
+  }
+
+  @Test(expected = StatusNotFoundException.class)
+  public void keyboardStateUnsuccessfullTest() {
+    String result = "A F U C";
```

Comment:
result should probably be named statusString.
Within this test I cannot immediately see, why it should fail (e.g. add a // <--- keyboard state A is illegal)
The test itself is not testing whether the keyboardStatus is illegal, but whether any of the values is illegal. This is a bit misleading ...
Given/When/Then is not visible (if though trivial)